src/Me/Me.tsx

```diff
@@ -1,34 +1,26 @@
 import React from 'react';
-import { useRouter } from 'next/router';
 import { ToastContainer } from 'react-toastify';
 import 'react-toastify/dist/ReactToastify.css';
 import styled from 'styled-components/macro';
+import { Helmet } from 'react-helmet';
+
 import Header from './Header/Header';
 import Main from './Main';
 import Navbar from './Navigation/Navbar';
-// import Home from './Routes/Home';
-// import MentorshipRequests from '../Me/MentorshipRequests';
 import { GlobalStyle } from './styles/global';
 import { desktop } from './styles/shared/devices';
-// import { AuthorizationRoute } from '../CustomRoutes/AuthorizedRoute';
-import { Helmet } from 'react-helmet';
-
-// const Admin = React.lazy(() =>
-//   import(/* webpackChunkName: "Admin" */ './Routes/Admin')
-// );
+import { isSsr } from '../helpers/ssr';
 
 
 
 const Me = (props: any) => {
   const { children, title } = props
-  const { pathname } = useRouter();
   
   // Ensure we're actually in a browser before rendering the component
-  if (typeof window === 'undefined') {
+  if (!isSsr()) {
```

Comment:
```suggestion
  if (isSsr()) {
```
🤔
Also we probably can remove the comment above now.

Reply:
Good catch!